app/src/androidTest/java/com/plcoding/calorytracker/TrackerOverviewE2E.kt

```diff
@@ -0,0 +1,236 @@
+import com.plcoding.core.domain.model.UserInfo
+import com.plcoding.core.domain.preferences.Preferences
+import com.plcoding.core.domain.use_case.FilterOutDigits
+import com.plcoding.tracker.tracker_domain.model.TrackableFood
+import com.plcoding.tracker.tracker_domain.use_case.*
+import com.plcoding.tracker_presentation.search.SearchScreen
+import com.plcoding.tracker_presentation.search.SearchViewModel
+import com.plcoding.tracker_presentation.tracker_overview.TrackerOverviewScreen
+import com.plcoding.tracker_presentation.tracker_overview.TrackerOverviewViewModel
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import io.mockk.InternalPlatformDsl.toStr
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import javax.inject.Inject
+import kotlin.math.roundToInt
+
+@HiltAndroidTest
+class TrackerOverviewE2E {
+
+    @get:Rule
+    val hiltRule = HiltAndroidRule(this)
+
+    @get:Rule
+    val composeRule = createAndroidComposeRule<MainActivity>()
+
+    private lateinit var repositoryFake: TrackerRepositoryFake
+    private lateinit var trackerUseCases: TrackerUseCases
+    private lateinit var preferences: Preferences
+    private lateinit var trackerOverviewViewModel: TrackerOverviewViewModel
+    private lateinit var searchViewModel: SearchViewModel
+
+    private lateinit var navController: NavHostController
+
+    @Before
+    fun setUp() {
+        preferences = mockk(relaxed = true)
+        every { preferences.loadUserInfo() } returns UserInfo(
+            gender = Gender.Male,
+            age = 20,
+            weight = 80f,
+            height = 180,
+            activityLevel = ActivityLevel.Medium,
+            goalType = GoalType.KeepWeight,
+            carbRatio = 0.4f,
+            proteinRatio = 0.3f,
+            fatRatio = 0.3f
+        )
+        repositoryFake = TrackerRepositoryFake()
+        trackerUseCases = TrackerUseCases(
+            trackFood = TrackFood(repositoryFake),
+            searchFood = SearchFood(repositoryFake),
+            getFoodsForDate = GetFoodsForDate(repositoryFake),
+            deleteTrackedFood = DeleteTrackedFood(repositoryFake),
+            calculateMealNutrients = CalculateMealNutrients(preferences)
+        )
+        trackerOverviewViewModel = TrackerOverviewViewModel(
+            preferences = preferences,
+            trackerUseCases = trackerUseCases
+        )
+        searchViewModel = SearchViewModel(
+            trackerUseCases = trackerUseCases,
+            filterOutDigits = FilterOutDigits()
+        )
+        composeRule.setContent {
+            CaloryTrackerTheme {
+                val scaffoldState = rememberScaffoldState()
+                navController = rememberNavController()
+                Scaffold(
+                    modifier = Modifier.fillMaxSize(),
+                    scaffoldState = scaffoldState
+                ) { padding ->
```

Comment:
> Since Compose 1.2.0 it's required to use padding parameter, passed into Scaffold content composable. You should apply it to the topmost container/view in content

https://stackoverflow.com/questions/72084865/content-padding-parameter-it-is-not-used